airgun/helpers/base.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any, Dict, List, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from airgun.entities.base import BaseEntity
+
+
+class BaseEntityHelper:
+
+    def __init__(self, entity):
+        # type: (BaseEntity) -> None
+        self._entity = entity
+
+    @property
+    def entity(self):
+        return self._entity
+
+    def form_fill_reader(self, navigation_name, navigation_kwargs=None, values=None,
+                         read_widget_names=None):
+        # type: (str, Dict, Dict[str, Any], List[str]) -> Dict[str, Any]
+        """Navigate to a form with registered name navigation_name and with kwargs
+        navigation_kwargs, fill the form with values and read read_widget_names if supplied
+        otherwise read the fields in values.
```

Comment:
that will be nice to provide one example in docstring, like to fill 2 widgets and read 2 widgets in one call

Reply:
added an example